scripts/example_NASA_FIRMS.py

```diff
@@ -1,10 +1,34 @@
-from pyronear_ds.datasets import NASAFIRMS
+from pyronear_ds.datasets import NASAFIRMS, NOAAWeather
+from pyronear_ds.datasets.datasets_mergers import merge_datasets_by_closest_weather_station
 
 
 def main(args):
-
+    weather = NOAAWeather(args.weather)
     nasa_firms = NASAFIRMS(args.nasa_firms, args.nasa_firms_type)
-    print(nasa_firms)
+    print(weather.shape)
+    print(nasa_firms.shape)
+
+    merged_data = merge_datasets_by_closest_weather_station(weather, 'DATE', nasa_firms, 'acq_date')
```

Comment:
maybe it's better to have a merging option, either by closest weather station or by department ?

Reply:
Then the user can change the function non ? Or maybe we should add a parsing argument ? That could be nice indeed

Reply:
yes I meant a parsing argument or a parameter in the inputs of the main function to change the merge type

Reply:
done, let me know if it's okay for you